Update Moveables_CN.txt

AIMEIXIZI opened the pull request:
我这显示无法自动合并是什么意思啊

Daybreakaxl commented on CN/Moveables_CN.txt:
```diff
@@ -134,19 +134,19 @@ Moveables_CN = {
+    Dark_Fancy_Wardrobe = "深色精致衣柜",
+    Dark_Green_Rug = "深绿色地毯",
+    Dark_Grey_Carpet = "深灰色地毯",
     Dark_Grey_Rug = "Dark Grey Rug",
```
Dark_Grey_Rug = "深灰色小地毯"

Mishasama commented:
> 我这显示无法自动合并是什么意思啊

证明你源文件版本比合并目标的文件版本要旧，你需要检查并解决内容冲突后才能进行合并。
详细可在Web页面里查看并操作，自己摸索一下吧，我英语0级都没问题，相信你也可以的。😉

AIMEIXIZI commented:
ok我试试

Misha <notifications@github.com>于2020年4月18日 周六下午8:04写道：

> 我这显示无法自动合并是什么意思啊
>
> 证明你源文件版本比合并目标的文件版本要旧，你需要检查并解决内容冲突后才能进行合并。
> 详细可在Web页面里查看并操作，自己摸索一下吧，我英语0级都没问题，相信你也可以的。😉
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#issuecomment-615915229>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOV7SB5EUWQGXIMK72C3KA3RNHTULANCNFSM4MLCT75A>
> .
>

Commit:
Updated MakeUp_EN.txt

Typo corrections

Commit:
41.37

Update für 41.37.
Hauptsächlich MakeUp_de.txt

Movables fehlen noch.

nsKpz committed:
Merge pull request #239 from TheIndieStone/DE

41.37

Commit:
Updated Credits file

Discarded "Translated by.txt" - no longer used.

narrnika committed:
Update news for 41.37

https://projectzomboid.com/blog/2020/05/urban-terrorz/

narrnika committed:
Fixes

narrnika committed:
More fixes

PrincipeAgustin committed:
Finish Sandbox_AR

- Finish Sandbox_AR.
- Add a 41.37 UI_AR line.

PrincipeAgustin committed:
Spelling changes

Faalagorn committed:
4 fixes for movable objects names

Fixed two typos and changed two signs name to closely match what's in them

geras-PL committed:
update

narrnika committed:
Update

Added GUI_Game_Action_* for Gamepad-Build
New line for 41.37

Faalagorn committed:
Refridgeration -> Refrigeration

https://theindiestone.com/forums/index.php?/topic/28671-typo-refrigeration-not-refridgeration-no-d/

PrincipeAgustin committed:
Fixed AR

- Add some recipies.
- Change items names.
- Some changes in context menu.

PrincipeAgustin committed:
Fixes ES_AR

- Fixed vehicle traduccion errors.

nsKpz committed:
Merge pull request #243 from PrincipeAgustin/master

Update ES_AR

nsKpz committed:
Merge pull request #242 from narrnika/master

Fixes and news for 41.37 (RU)

narrnika committed:
Update to 41.37.01 for Russian

ditoseadio committed:
Build 41.38 ready

PrincipeAgustin committed:
41.38 and recipies

Commit:
Korean Translation Update

By SeaMustard

nsKpz committed:
Merge pull request #244 from Faalagorn/patch-10

4 fixes for movable objects names

nsKpz committed:
Merge pull request #211 from Faalagorn/patch-6

Various updates to sandbox options

nsKpz committed:
Merge pull request #207 from Faalagorn/patch-4

Made all moodles title case

nsKpz committed:
Merge pull request #206 from Faalagorn/patch-3

Various IG UI fixes

nsKpz committed:
Merge pull request #205 from Faalagorn/patch-2

Various fixes for context menu names

nsKpz committed:
Merge pull request #245 from narrnika/master

Update for Russian

nsKpz committed:
Merge pull request #246 from TheIndieStone/ditoseadio-patch-5

Build 41.38 ready

nsKpz committed:
Merge pull request #247 from PrincipeAgustin/master

41.38 and recipies

Commit:
Revert "Merge pull request #205 from Faalagorn/patch-2"

This reverts commit 92c5bc971e2659639a73538e6084b788102dbc09, reversing
changes made to 0222b2869a3a18591bedb939268cce8e2215b9f6.

dfr1238 committed:
Merge pull request #1 from TheIndieStone/master

Merge From Origin

Commit:
Catch-Up for UI_EN.txt

Previous update to the file was behind the game version.

Incorporated missing lines as well as some feedback from Faalgorn.

Commit:
Update UI_DE.txt

nsKpz committed:
Merge pull request #248 from TheIndieStone/_de

Update UI_DE.txt

Commit:
Update UI_EN.txt

Clarifcation for VAC server option tooltip. This is a leftover from a preliminary multiplayer version early on; VAC was never used by PZ.

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

Mishasama committed:
41.37

+ MakeUp
+ New strings for 41.37
* Some polish

geras-PL committed:
41.37

narrnika committed:
Update to 41.37.02 for Russian

AIMEIXIZI committed:
Update Moveables_CN.txt

AIMEIXIZI committed:
Update Stash_CN.txt

Commit:
Updated Items_empty.txt

Removed Duplicates.
Removed obsolete items.
Added missing items.
Sorted alphabetically.

Commit:
Updated Recipe_empty.txt

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -147,64 +147,64 @@ Moveables_CN = {
+    Diner_Counter = "小餐馆柜台",
+    Diner_Floor = "小餐馆地板",
+    Diner_Neon_Sign = "小餐馆霓虹灯招牌",
     Dixie_Sign = "Dixie Sign",
```
```suggestion
    Dixie_Sign = "迪克西标志",
```

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -147,64 +147,64 @@ Moveables_CN = {
     Empty_Mall_Decoration = "Empty Mall Decoration",
-    Empty_Pallet = "Empty Pallet",
+    Empty_Pallet = "空托盘",
     Enigma_Books_Sign = "Enigma Books Sign",
```
```suggestion
    Enigma_Books_Sign = "谜语书店标志",
```

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -147,64 +147,64 @@ Moveables_CN = {
+    Exceptionally_crafted_Chair = "极好的手工椅",
+    Exit_Sign = "出口标志",
+    Eye_Test_Poster = "眼科测试海报",
     Family_Fashion_Sign = "Family Fashion Sign",
```
```suggestion
    Family_Fashion_Sign = "家居时装标志",
```

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -147,64 +147,64 @@ Moveables_CN = {
+    Fancy_Counter = "精致柜台",
+    Fancy_Dark_Table = "精致深色桌子",
+    Fancy_Drawers = "精致抽屉",
     Fancy_Floating_Counter = "Fancy Floating Counter",
```
```suggestion
    Fancy_Floating_Counter = "精致流动柜台",
```

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -147,64 +147,64 @@ Moveables_CN = {
+    Fancy_Outdoor_Lamp = "精致室外灯",
+    Fancy_Outdoors_Bench = "精致户外长凳",
+    Fancy_Table = "精致桌",
     Fancy_Toilet = "Fancy Toilet",
```
```suggestion
    Fancy_Toilet = "精致厕所",
```

Mishasama reviewed (dismissed):
还行😂
物品数量太多我没空进游戏一一核对翻译是否正确了……
如果条件允许的话希望你能替我对着游戏里的物品来以实物为准，因为我印象里第187行的“眼科”海报应该是视力测试表，但我不是很确定………

AIMEIXIZI commented:
对对对，应该是视力测试表，我之前翻译的时候硬是想不起来这个词
其他的词我大概都会进游戏看看（如果我有时间的话）有些奇怪的东西我还是会进游戏看看的
对了，这里面还有些关于品牌的我不知道怎么翻译好

Misha <notifications@github.com> 于2020年5月18日周一 下午8:48写道：

> *@Mishasama* approved this pull request.
>
> 还行😂
> 物品数量太多我没空进游戏一一核对翻译是否正确了……
> 如果条件允许的话希望你能替我对着游戏里的物品来以实物为准，因为我印象里第187行的“眼科”海报应该是视力测试表，但我不是很确定………
> ------------------------------
>
> In CN/Moveables_CN.txt
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#discussion_r426817717>
> :
>
> > +    Dark_Purple_Rug = "深紫色地毯",
>
> +    Dark_Red_Carpet = "深红色地毯",
>
> +    Dark_Round_Table = "深色圆桌",
>
> +    Dark_School_Bench = "深色学校长椅",
>
> +    Dark_Wall_Lamp = "深色壁灯",
>
> +    Dark_Wall_Shelves = "深色墙架",
>
> +    Dark_Wooden_Chair = "深色木制椅",
>
> +    Darkblue_Modern_Chair = "深蓝色现代椅",
>
> +    Decorated_Altar = "装饰用祭坛",
>
> +    Deluxe_Shower = "高级淋浴器",
>
> +    Dentist_Patient_Chair = "牙医诊所椅",
>
> +    Desktop_Computer = "台式电脑",
>
> +    Diner_Corner_Counter = "小餐馆转角柜台",
>
> +    Diner_Counter = "小餐馆柜台",
>
> +    Diner_Floor = "小餐馆地板",
>
> +    Diner_Neon_Sign = "小餐馆霓虹灯招牌",
>
>      Dixie_Sign = "Dixie Sign",
>
>
> ⬇️ Suggested change
>
> -    Dixie_Sign = "Dixie Sign",
>
> +    Dixie_Sign = "迪克西标志",
>
>
> ------------------------------
>
> In CN/Moveables_CN.txt
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#discussion_r426821323>
> :
>
> >      Empty_Mall_Decoration = "Empty Mall Decoration",
>
> -    Empty_Pallet = "Empty Pallet",
>
> +    Empty_Pallet = "空托盘",
>
>      Enigma_Books_Sign = "Enigma Books Sign",
>
>
> ⬇️ Suggested change
>
> -    Enigma_Books_Sign = "Enigma Books Sign",
>
> +    Enigma_Books_Sign = "谜语书店标志",
>
>
> ------------------------------
>
> In CN/Moveables_CN.txt
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#discussion_r426822260>
> :
>
> >      Enigma_Books_Sign = "Enigma Books Sign",
>
> -    Exceptional_Large_Table = "Exceptional Large Table",
>
> -    Exceptional_Table = "Exceptional Table",
>
> -    Exceptionally_crafted_Chair = "Exceptionally crafted Chair",
>
> -    Exit_Sign = "Exit Sign",
>
> -    Eye_Test_Poster = "Eye Test Poster",
>
> +    Exceptional_Large_Table = "极好的大桌子",
>
> +    Exceptional_Table = "极好的桌子",
>
> +    Exceptionally_crafted_Chair = "极好的手工椅",
>
> +    Exit_Sign = "出口标志",
>
> +    Eye_Test_Poster = "眼科测试海报",
>
>      Family_Fashion_Sign = "Family Fashion Sign",
>
>
> ⬇️ Suggested change
>
> -    Family_Fashion_Sign = "Family Fashion Sign",
>
> +    Family_Fashion_Sign = "家居时装标志",
>
>
> ------------------------------
>
> In CN/Moveables_CN.txt
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#discussion_r426823002>
> :
>
> >      Family_Fashion_Sign = "Family Fashion Sign",
>
> -    Fancy_Bed = "Fancy Bed",
>
> -    Fancy_Blue_Rug = "Fancy Blue Rug",
>
> -    Fancy_Brown_Rug = "Fancy Brown Rug",
>
> -    Fancy_Counter = "Fancy Counter",
>
> -    Fancy_Dark_Table = "Fancy Dark Table",
>
> -    Fancy_Drawers = "Fancy Drawers",
>
> +    Fancy_Bed = "精致床",
>
> +    Fancy_Blue_Rug = "精致蓝色地毯",
>
> +    Fancy_Brown_Rug = "精致棕色地毯",
>
> +    Fancy_Counter = "精致柜台",
>
> +    Fancy_Dark_Table = "精致深色桌子",
>
> +    Fancy_Drawers = "精致抽屉",
>
>      Fancy_Floating_Counter = "Fancy Floating Counter",
>
>
> ⬇️ Suggested change
>
> -    Fancy_Floating_Counter = "Fancy Floating Counter",
>
> +    Fancy_Floating_Counter = "精致流动柜台",
>
>
> ------------------------------
>
> In CN/Moveables_CN.txt
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#discussion_r426823302>
> :
>
> > -    Fancy_Oak_Drawers = "Fancy Oak Drawers",
>
> -    Fancy_Oak_Wardrobe = "Fancy Oak Wardrobe",
>
> -    Fancy_Orange_Rug = "Fancy Orange Rug",
>
> -    Fancy_Outdoor_Lamp = "Fancy Outdoor Lamp",
>
> -    Fancy_Outdoors_Bench = "Fancy Outdoors Bench",
>
> -    Fancy_Table = "Fancy Table",
>
> +    Fancy_Green_Rug = "精致绿色地毯",
>
> +    Fancy_Hanging_Sink = "精致水槽",
>
> +    Fancy_Low_Glass_Table = "精致玻璃矮桌",
>
> +    Fancy_Low_Table = "精致矮桌",
>
> +    Fancy_Oak_Drawers = "精致橡木抽屉",
>
> +    Fancy_Oak_Wardrobe = "精致橡木衣柜",
>
> +    Fancy_Orange_Rug = "精致橙色地毯",
>
> +    Fancy_Outdoor_Lamp = "精致室外灯",
>
> +    Fancy_Outdoors_Bench = "精致户外长凳",
>
> +    Fancy_Table = "精致桌",
>
>      Fancy_Toilet = "Fancy Toilet",
>
>
> ⬇️ Suggested change
>
> -    Fancy_Toilet = "Fancy Toilet",
>
> +    Fancy_Toilet = "精致厕所",
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Mishasama/ProjectZomboidTranslations/pull/58#pullrequestreview-413839247>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOV7SB3EFHSKU5WW7642HQTRSF7JTANCNFSM4MLCT75A>
> .
>

Mishasama commented on CN/Moveables_CN.txt:
```diff
@@ -134,77 +134,77 @@ Moveables_CN = {
+    Exceptional_Table = "极好的桌子",
+    Exceptionally_crafted_Chair = "极好的手工椅",
+    Exit_Sign = "出口标志",
+    Eye_Test_Poster = "眼科测试海报",
```
```suggestion
    Eye_Test_Poster = "视力测试表",
```

AIMEIXIZI committed:
Update Moveables_CN.txt

Mishasama reviewed (commented):
不知道的东西可以在游戏里对着物品截个图，然后标记一下是哪一行，整理出一个文件丢我，我来处理那些10级英语😂
我在评论里提出的那些更改你可以直接在评论区里逐个整合，或者去Files changed里批量整合，这样就不用你再手动改省点事。

Commit:
Added title.txt for spawnpoint title translation.

Added title.txt for languages with folders for the spawnpoints.

Translations for map titles can now be specified by editing the title.txt in the same folder as description.txt.

For example, media\lua\shared\Translate\FR\Muldraugh, KY\title.txt.
The file should only contain a single line with the translated map name.
The file is empty by default, displaying the original name.

narrnika committed:
Fixes

-debugtranslation
- AEBS_Power

Faalagorn committed:
Update Recipe_empty.txt

* Incorporates fixes from https://theindiestone.com/forums/index.php?/topic/24011-minor-correction-to-item-names-in-txt-files/&tab=comments#comment-273074 and #224
* Fixes to match #251 
* Various other fixes

John2022 committed:
FR files update via files upload 2020-05-20

John2022 committed:
FR files update via files upload 2020-05-20

nsKpz committed:
Merge pull request #250 from narrnika/master

Update for Russian

nsKpz committed:
Merge pull request #249 from TheIndieStone/polish

Polish 41.37

narrnika committed:
Merge pull request #4 from TheIndieStone/master

pull changes from main repository

narrnika committed:
Update to 41.37.03 for Russian

- titles for cities
- last stand 2

John2022 committed:
FR files update via files upload 2020.05.21

nsKpz committed:
Merge pull request #253 from narrnika/master

Update for Russian

Commit:
Added ItemName_XX.txt

ItemName_XX.txt added to languages that did not already have one.

Translations should only happen to these from now on to prevent translations from breaking when English item names are changed.

Languages that already had an updated ItemName_XX.txt did not have their Items_XX.txt converted, so any differences will have to be added manually by translators.

Added infotext to Items_XX.txt files that Items_XX.txt should no longer be used.

nsKpz committed:
Merge pull request #254 from TheIndieStone/ItemNameChange

Added ItemName_XX.txt

Commit:
Correction of the ".556" ammunition mistake

Shinyshark1 committed:
Add files via upload

Updated: Translation for the Dutch Language

Shinyshark1 committed:
Merge pull request #1 from MaximvanMaanen/Dutch-Translation

Dutch Translation

John2022 committed:
FR files update via files upload 2020.05.22

Commit:
Updated ItemName_EN.txt

Applied plenty of feedback from the great Faalgorn. (#251)

Commit:
Update ItemName_EN.txt

Added colour to medical caps and masks

Commit:
Updated ContextMenu_EN.txt

With feedback from Faalgorn

nsKpz committed:
Merge pull request #255 from MaximvanMaanen/master

Updated the Dutch Translation files

Commit:
Updated UI_EN.txt

Removed lines only relevant to the demo, as there is no PZ demo anymore.

Implemented feedback from Faalagorn (finally getting his name right)

John2022 committed:
FR files update via files upload 2020.05.23

dfr1238 committed:
Merge pull request #2 from TheIndieStone/master

Merge Original

dfr1238 committed:
Update TChinese Translate

-Update old Trad Chinese Translate, updated to 41 build.
-Fix all missing word.
-Fix some word the term, make it part from SChinese, remove china term.
-Make all punctuation use FULL-CHARACTER for good reading.
-Add and update Map desc and title

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
Add files via upload

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.24

John2022 committed:
FR files update via files upload 2020.05.26

Commit:
Update from #256

With compatibility changes to encoding

nsKpz committed:
Merge pull request #257 from dfr1238/master

Update/fix TChinese old translations

Commit:
Update ItemName_PTBR.txt

UTF-8 to ANSI

Commit:
Update ItemName_EN.txt

Minor edits. Added dashes to Canoe Paddle and Choke tube variations ; Feedback from Faalagorn

John2022 committed:
FR files update via files upload 2020.05.26

Commit:
Update UI_EN.txt

nsKpz committed:
Merge pull request #258 from TheIndieStone/de

Change to UI_EN

John2022 committed:
FR files update via files upload 2020.05.27

Commit:
Update ItemName_PTBR.txt

Changes from #259

Commit:
Update ItemName_EN.txt

Item names adjusted to correct errors after convo in #252

Commit:
Update DynamicRadio_EN.txt

typo correction

Commit:
Update ItemName_DE.txt

John2022 committed:
FR files update via files upload 2020.05.29

John2022 committed:
FR files update via files upload 2020.05.29

John2022 committed:
FR files update via files upload 2020.05.29

nsKpz committed:
Merge branch 'master' into master

nsKpz committed:
Merge pull request #260 from John2022/master

French translation update

Commit:
FR Update

Removed Items_FR.txt from root folder.
Covernted FR/Items_FR.txt to ItemName_FR.txt

Commit:
Update Items_FR.txt

Encoding fix

narrnika committed:
Merge pull request #5 from TheIndieStone/master

pull changes from main repository

Faalagorn committed:
Update Recipe_empty.txt

Mishasama committed:
Revert "Merge branch 'Mishasama-WIP-patch-2' into master"

This reverts commit 2157b2a0d406a3cf5f07213deeff560cc913ff03, reversing
changes made to 9a386da29f141404d04285145506a42f16268f16.

Mishasama committed:
Update ItemName_EN.txt

Fixes #76

Mishasama committed:
Update UI_EN.txt

Fixes #76

Mishasama committed:
Merge pull request #76 from TheIndieStone/master

Update

Mishasama committed:
Merge branch 'Mishasama-WIP-patch-2' into master

Mishasama committed:
Merge pull request #77 from Mishasama/master

update

nsKpz committed:
Merge pull request #252 from Faalagorn/patch-3

Update Recipe_empty.txt

Commit:
Update Recipes_EN.txt

Commit:
41.38

Commit:
Korean Update

Shinyshark1 committed:
Updated Dutch Translations

Updated the Dutch Translations for 01/06/2020

Commit:
Update IG_UI_EN.txt for 41.38

Added hairstyles
Added strings for radial reloading menu

nsKpz committed:
Merge pull request #261 from MaximvanMaanen/master

Updated Dutch Translations

Commit:
Delete NL_englishcache_v1.properties

Commit:
Deletion of .properties files

These are created when using TranslationZed but aren't needed for the translations themselves.

Commit:
Update IG_UI_DE.txt

narrnika committed:
Update to 41.37.05 for Russian

Commit:
Update ContextMenu_EN.txt

Added new composter context menu strings

narrnika committed:
Update and fixing

nsKpz committed:
Merge pull request #262 from narrnika/master

Update for Russian

Commit:
Updated 41.38

Hammerbund committed:
Add files via upload

Hammerbund committed:
Add files via upload

Commit:
Update IG_UI_EN.txt

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

nsKpz committed:
Merge pull request #264 from Hammerbund/Test

Verbesserungen

Faalagorn committed:
Unified capitalization of "XP"

Faalagorn committed:
Update IG_UI_EN.txt

Faalagorn committed:
Update UI_EN.txt

Faalagorn committed:
UI_EN: Unified "requires restart" tooltips

Both of these options appear under "Display" option section, for "Borderless Window" and "Multi-core", I adjusted them to match existing descriptions. On one hand the same string should only be used in both cases as these serve the same purpose – one less string to store and translate, but on the other hand maybe some unique tooltips could be added to those options explaining what they do in future.

Faalagorn committed:
EN: Minor changes to sandbox settings

* unified metagame spelling and removed unneeded big letter from "event"

Faalagorn committed:
Tooltip_EN: unified "click" spelling

Changed left-click to "left click" to match other spellings currently

Faalagorn committed:
Tooltip_EN: Removed dot from trunk to match others

Removed dot from "Trunk is locked" and "Trunk is unlocked" to match other car tooltips. However, the tooltips themselves might get reworked somewhere down the line, as they are not consistent with each other.

Faalagorn committed:
IG_UI_EN: Minor fixes for foraging menu

Removed dot and fixed casing in foraging menu

Faalagorn committed:
EN: Bullet → Round

Changed bullets to rounds where applicable, i.e. when a cartridge is complete with case, gunpowder and primer and bullet it's typically called round. I left the bullets in molds though, as you can only mold that part of cartridge (round) only.

Faalagorn committed:
Update Recipes_EN.txt

Fixed missing "mm" while at it

Commit:
41.38

Added progress bar accessibility option.
Bars are going to be hidden on actions that have a visible animation.

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

nsKpz committed:
Merge pull request #265 from Faalagorn/patch-2

EN: Unified capitalization of "XP"

nsKpz committed:
Merge pull request #266 from Faalagorn/patch-3

UI_EN: Unified "requires restart" tooltips

nsKpz committed:
Merge pull request #270 from Faalagorn/patch-10

Tooltip_EN: unified "click" spelling

nsKpz committed:
Merge pull request #273 from Faalagorn/patch-13

IG_UI_EN: Minor fixes for foraging menu

Hammerbund committed:
Add files via upload

Faalagorn committed:
Update Recipes_EN.txt

Speciam → Special

nsKpz committed:
Merge pull request #274 from Faalagorn/patch-14

EN: Bullet → Round

nsKpz committed:
Merge pull request #272 from Faalagorn/patch-12

Tooltip_EN: Removed dot from trunk to match others

Hammerbund committed:
Update Recipes_DE.txt

Hammerbund committed:
Update UI_DE.txt

nsKpz committed:
Merge pull request #277 from Hammerbund/master

Various changes and additions

Commit:
Update to german files

Updated credits
Corrected wrong encodings for compatibility
Added "welcome to 41" blurlb

Commit:
41.38

Added items new in 41.38

Commit:
Updated english base files

Shinyshark1 committed:
Updated Dutch Translations V41.38

Faalagorn committed:
PizzaWhirled → Pizza Whirled

PizzaWhirled Server Apron → Pizza Whirled Server Apron

Faalagorn committed:
Update ItemName_empty.txt

Hammerbund committed:
Add files via upload

narrnika committed:
Corrections

EvolvedRecipes
Bottles
Carpentry & Metalwork ContextMenu
Movables

narrnika committed:
Update to 41.37.07 for Russian

Update without tutorial
News for 41.38
Items legacy

Faalagorn committed:
Whiskey → Bourbon

Fixes https://theindiestone.com/forums/index.php?/topic/28930-immersion-breaking-bug-a-letter-to-the-devs/

Faalagorn committed:
Update ItemName_empty.txt

Faalagorn committed:
Update Recipes_EN.txt

Faalagorn committed:
Update Recipes_empty.txt

nsKpz committed:
Merge pull request #279 from Faalagorn/patch-2

PizzaWhirled → Pizza Whirled

nsKpz committed:
Merge pull request #268 from Faalagorn/patch-6

EN: Minor changes to sandbox settings

Commit:
Delete NL_englishcache_v1.properties

nsKpz committed:
Merge pull request #283 from TheIndieStone/pr/278

Pr/278

Commit:
Moveables_DE

Implemented translations from #280 to Moveables_DE and ContextMenu_DE

Corrected some store names to reflect their brand name

Commit:
Update Moveables_EN.txt

Changed some signs to reflect their brand names

nsKpz committed:
Merge pull request #280 from Hammerbund/master

Add files via upload

nsKpz committed:
Merge pull request #281 from narrnika/master

Update for Russian

Commit:
Update Moveables_DE.txt

Sorted alphabetically
Added a couple translations

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

Commit:
DE Update

Various additions and corrections

Hammerbund committed:
Add files via upload

Hammerbund committed:
Add files via upload

Hammerbund committed:
Add files via upload

Hammerbund committed:
Delete ContextMenu_DE.txt

nsKpz committed:
Merge pull request #284 from Hammerbund/master

Verbesserungen

Hammerbund committed:
Update ItemName_DE.txt

nsKpz committed:
Merge pull request #282 from Faalagorn/patch-3

Whiskey → Bourbon

Commit:
Small update to be in line with 41.38

Updated tutorial texts ***STILL NOT FINAL - I do not recommend translating these yet***

Added separate IGUI_ClothingName_XXX entries separate from Tooltip_clothing_XXX (Wet, Dirty, Soaked, etc)

Clothing Type now translatable through UI_ClothingTextureType

Faalagorn committed:
UI_EN: Minor unification for loading text

* removed dot from game booting string
* texturepack → texture pack
* unified casing

Faalagorn committed:
PL: Fixes for Polish translation

Faalagorn committed:
README: Add TranslationZed and local testing notes

* Added note about TranslationZed and testing translations locally

narrnika committed:
Update to 41.37.08 for Russian

Add new lines
Fix gun-menu

nsKpz committed:
Merge pull request #285 from Hammerbund/master

Update ItemName_DE.txt

nsKpz committed:
Merge pull request #291 from narrnika/master

Update for Russian

nsKpz committed:
Merge pull request #286 from Faalagorn/patch-1

UI_EN: Minor unification for loading text

nsKpz committed:
Merge pull request #290 from Faalagorn/patch-3

README: Add TranslationZed and local testing notes

nsKpz committed:
Update README.md

Hammerbund committed:
Add files via upload

Hammerbund committed:
Merge pull request #1 from Hammerbund/Hammerbund-Test

Verbesserung

Commit:
Nothing new, minor changes

Changed "New Roof Hiding" to "Roof Hiding" as it certainly isn't new, anymore.

Some casing fixes based on Faalagorn feedback

Changed DE/ItemName_DE encoding for font compatibility

Commit:
Updated tutorial texts

nsKpz committed:
Merge pull request #292 from Hammerbund/master

Verbesserung

nsKpz committed:
Merge pull request #289 from Faalagorn/polishfixes1

PL: Fixes for Polish translation

Commit:
Updated tutorial texts

Ready for translation

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

Shinyshark1 committed:
Dutch Translation Tutorial

Translation for the Dutch Tutorial.

ditoseadio committed:
Build 41.38 Spanish Ready

Commit:
Delete NL_englishcache_v1.properties

nsKpz committed:
Merge pull request #295 from TheIndieStone/pr/293

Pr/293

nsKpz committed:
Merge pull request #294 from TheIndieStone/ditoseadio-patch-6

Build 41.38 Spanish Ready

Hammerbund committed:
Add files via upload

Hammerbund committed:
Merge pull request #1 from Hammerbund/Hammerbund-patch-3

Add files via upload

Hammerbund committed:
Add files via upload

Hammerbund committed:
Merge pull request #2 from Hammerbund/Hammerbund-patch-4

Add files via upload

narrnika committed:
Update to 41.37.09 for Russian

Update tutorial

Commit:
Update IG_UI_EN.txt

Changed line IGUI_Tutorial1_Fight8New

nsKpz committed:
Merge pull request #296 from Hammerbund/master

single line change

nsKpz committed:
Merge pull request #297 from narrnika/master

Update for Russian

Commit:
First draft of German tutorial

Commit:
Merge branch 'master' of https://github.com/TheIndieStone/ProjectZomboidTranslations

Mishasama committed:
Create News_CN.txt

+ 41.37 News

Mishasama committed:
Update Items_CN.txt

+ Cases

Mishasama committed:
Merge branch 'Mishasama-WIP-patch-2' of https://github.com/Mishasama/ProjectZomboidTranslations into Mishasama-WIP-patch-2

Mishasama committed:
Update by replacing

resolve conflicts

Mishasama committed:
Merge pull request #81 from TheIndieStone/master

update to 41.38

Mishasama committed:
41.38

Mishasama committed:
Merge branch 'Mishasama-WIP-patch-2' of https://github.com/Mishasama/ProjectZomboidTranslations into Mishasama-WIP-patch-2

Commit:
Updated Korean

via SeaMustard on Discord

Mishasama committed:
Update News_CN.txt

41.38

Stake2 committed:
Changed ContextMenu_Add_petrol text which was wrong

Changed ContextMenu_Add_petrol from "Deitar gasolina" to "Adicionar gasolina", which means add gasoline

nsKpz committed:
Merge pull request #298 from stake2/patch-1

Changed ContextMenu_Add_petrol text which was wrong

Commit:
Update ContextMenu_PT.txt

Encoding change for compatibility

Faalagorn committed:
Ball Peen Hammer → Ball-peen hammer

Seems to be more common spelling https://en.wikipedia.org/wiki/Ball-peen_hammer

Faalagorn committed:
Update ItemName_empty.txt

Faalagorn committed:
Update Recipes_empty.txt

Faalagorn committed:
Update Recipes_EN.txt

Faalagorn committed:
Update README.md

Some minor formatting/wording fixes for my previous rough commit.

Faalagorn committed:
Motel Counter → White Counter

It is used in some houses too, e.g. https://map.projectzomboid.com/#10912x9997

narrnika committed:
Fixes for Russian

narrnika committed:
Update to 41.39.01 for Russian

Added jewelry

nsKpz committed:
Merge pull request #299 from Faalagorn/patch-1

Ball Peen Hammer → Ball-peen Hammer

nsKpz committed:
Merge pull request #300 from Faalagorn/patch-3

Update README.md

nsKpz committed:
Merge pull request #303 from narrnika/master

Update for Russian

Faalagorn committed:
Raw Axe → Stone Axe

The recipe and other items use "Stone" name instead. Not sure why it was renamed to "Raw" at one point?

Faalagorn committed:
Update ItemName_empty.txt

Commit:
New Items for 41.40

Commit:
Inclusion of language.txt files

The language.txt specifies the name and character encoding for the respective language.
Old "Language" enum is now a class.  This shouldn't break mods (unless they were calling an unimplemented method).

This change was made so the hard-coded list of languages can be removed in order to allow new languages to be added without code-side changes.

nsKpz committed:
Merge pull request #302 from Faalagorn/patch-5

Motel Counter → White Counter

nsKpz committed:
Merge pull request #304 from Faalagorn/patch-1

Raw Axe → Stone Axe

Mishasama committed:
Up to 41.38

1. 新增除教程文本外的所有新内容的汉化；
2. 根据官方最新文本调整、润色大部分旧有内容；
3. 新增部分家具汉化；
4. 放弃了旧Items文件，正式迁移到新的通用ItemName文件；
5. 扶两位协力汉化的群友上墙！

Mishasama committed:
Update News to 41.39

Mishasama committed:
Merge pull request #85 from TheIndieStone/master

up to 41.40

Mishasama committed:
Update to 41.40

1. New Items for 41.40
https://github.com/Mishasama/ProjectZomboidTranslations/commit/360e365a59ad939482b3d2921bf2ac73b2a91abd
2. Inclusion of language.txt files …
https://github.com/Mishasama/ProjectZomboidTranslations/commit/159a9f005274a265d4caa484e5eae14aca345d00

Mishasama committed:
Merge branch 'Mishasama-WIP-patch-2' into AIMEIXIZI
